public/js/scripts.js

```diff
@@ -196,7 +196,24 @@ function toggleQuestionForm() {
 	    $("#newQuestionTextarea").val("");
 	}
 }
-
+//shows/hides feedback container
+function togglefeedbackForm() {
```

Comment:
In the recent design change pull request, I added a submit feedback button to the footer. We will want to map this function to that button eventually.